qdrant-landing/content/documentation/quickstart.md

```diff
@@ -409,34 +387,82 @@ ScoredPoint(id=3, version=0, score=1.208, payload={"city": "Moscow"}, vector=Non
 SearchResponse {
     result: [
         ScoredPoint {
-            id: Some(PointId {
-                point_id_options: Some(Num(4)),
-            }),
-            payload: {},
+            id: Some(
+                PointId {
+                    point_id_options: Some(
+                        Num(
+                            4,
+                        ),
+                    ),
+                },
+            ),
+            payload: {
+                "city": Value {
+                    kind: Some(
+                        StringValue(
+                            "New York",
+                        ),
+                    ),
+                },
+            },
             score: 1.362,
-            version: 0,
+            version: 2,
             vectors: None,
+            shard_key: None,
+            order_value: None,
         },
         ScoredPoint {
-            id: Some(PointId {
-                point_id_options: Some(Num(1)),
-            }),
-            payload: {},
+            id: Some(
+                PointId {
+                    point_id_options: Some(
+                        Num(
+                            1,
+                        ),
+                    ),
+                },
+            ),
+            payload: {
+                "city": Value {
+                    kind: Some(
+                        StringValue(
+                            "Berlin",
+                        ),
+                    ),
+                },
+            },
             score: 1.273,
-            version: 0,
+            version: 2,
             vectors: None,
+            shard_key: None,
+            order_value: None,
         },
         ScoredPoint {
-            id: Some(PointId {
-                point_id_options: Some(Num(3)),
-            }),
-            payload: {},
+            id: Some(
+                PointId {
+                    point_id_options: Some(
+                        Num(
+                            3,
+                        ),
+                    ),
+                },
+            ),
+            payload: {
+                "city": Value {
+                    kind: Some(
+                        StringValue(
+                            "Moscow",
+                        ),
+                    ),
+                },
+            },
             score: 1.208,
-            version: 0,
+            version: 2,
             vectors: None,
+            shard_key: None,
+            order_value: None,
         },
     ],
-    time: 0.003635125,
+    time: 0.001829198,
 }
```

Comment:
We might want to represent this just as JSON instead. The typed variant is really quite big and verbose. We do the same in the client documentation.

What's you opinion on this?

Reply:
+1